server/routerlicious/packages/services-shared/src/healthCheckEndpoints.ts

```diff
@@ -0,0 +1,102 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { Router } from "express";
+import { StartupChecker } from "./startupChecker";
+import { Lumberjack } from "@fluidframework/server-services-telemetry";
+
+/**
+ * Readiness status of a service or functionality.
+ * @internal
+ */
+export interface IReadinessStatus {
```

Comment:
see other comment, but if this is in `services-core` it will be easier to import around other places